MSAL/src/instance/MSALDeviceInfoProvider.m

```diff
@@ -78,7 +87,7 @@ - (void)deviceInfoWithRequestParameters:(MSIDRequestParameters *)requestParamete
 
         if (!ssoExtensionRequest)
         {
-            completionBlock(nil, requestError);
+            deviceInfoCompletionHandler(nil, requestError);
```

Comment:
ssoExtensionRequest being nil is an internal critical error, we may want to keep it as it is :)
 `completionBlock(nil, requestError);`